qds_sdk/commands.py

```diff
@@ -87,10 +87,26 @@ def run(cls, **kwargs):
         Returns:
             Command object
         """
+        err_pointer, tmp_pointer, new_bytes = 0, 0, 0
         cmd = cls.create(**kwargs)
         while not Command.is_done(cmd.status):
```

Comment:
@yogesh2021 if the command is completed, then err_length/tmp_length are ignored by tapp and complete logs are returned.

@msumit maybe we can add an `if` clause after `while` loop that checks if command is completed then compare the log length of the previous api-call with the current one. If there is any difference between log-length (err_length + tmp_length), then print that difference of logs(difference is to be printed from err_file)?